mcs/build/tests.make

```diff
@@ -159,4 +190,43 @@ $(test_makefrag): $(test_response)
 
 endif
 
+
+ifdef HAVE_CS_XTESTS
+
+XTEST_HARNESS_PATH = $(topdir)/../external/xunit-binaries
+XTEST_HARNESS = $(XTEST_HARNESS_PATH)/xunit.console.exe
+XTEST_HARNESS_FLAGS := -noappdomain -noshadow -parallel none -nunit XTestResult-$(PROFILE).xml
+XTEST_TRAIT := -notrait category=failing -notrait category=nonnetcoreapptests -notrait Benchmark=true
```

Comment:
would be awesome to add support for `FIXTURE` and `TESTNAME`, should be easy by passing `-class` and `-method` to xunit.